Framework/PythonInterface/inc/MantidPythonInterface/kernel/DataServiceExporter.h

```diff
@@ -27,11 +27,38 @@
 #include "MantidPythonInterface/kernel/WeakPtr.h"
 
 #include <boost/python/class.hpp>
-#include <boost/python/list.hpp>
 #include <boost/python/extract.hpp>
+#include <boost/python/list.hpp>
+#include <boost/python/str.hpp>
 
 #include <set>
 
+/**
+ * Convert a python object to a string or throw an exception. This will convert
+ * unicode strings in python2 via utf8.
+ *
+ * THIS IS DUPLICATED FROM
```

Comment:
Why does this need to be duplicated?

Reply:
It was having some issue with the function not being available from `_api.so` even though other things in `Converters` were.